.github/workflows/test.yml

```diff
@@ -1,20 +1,18 @@
-name: build
+name: test
 
 on:
   push:
     branches:
     - "master"
   pull_request:
-  schedule:
```

Comment:
I don't think we need to be running this regularly, besides, if there is no activity in the repository in 90 days scheduled workflows get automatically disabled anyway.